src/WebJobs.Script.WebHost/Metrics/FlexConsumptionMetricsPublisher.cs

```diff
@@ -91,10 +91,19 @@ internal void Initialize()
             IsAlwaysReady = _environment.GetEnvironmentVariable(EnvironmentSettingNames.FunctionsAlwaysReadyInstance) == "1";
         }
 
-        internal async Task OnPublishMetrics()
+        internal async Task OnPublishMetrics(DateTime now)
```

Comment:
tiny tiny nit: always prefer `DateTimeOffset` over `DateTime`.

Reply:
Gonna leave this as is, thanks.